### PR TITLE
DT-5977 Varely config

### PR DESCRIPTION
**Pull request by sharhio**
## Proposed Changes

  - Varely real-time data
  - Rauma data to Varely
  - Varely datat to Matka

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/configurations/config.matka.js [hunk 1]
```diff
@@ -510,6 +511,11 @@ export default {
       sv: 'Raseborg',
       en: 'Raasepori',
     },
+    VARELY: {
+      fi: 'Seutu+',
+      sv: 'Seutu+',
+      en: 'Seutu+',
+    },
```
Comment (optionsome) [inline app/configurations/config.matka.js]: Not quite sure what to call this but probably Varsinais-Suomi. Seutu+ is more than this data set (and it sounds slightly weird if you are not aware of what it is and you are interested in alerts around Rovaniemi region for example :D).

File: app/configurations/config.varely.js [hunk 2]
```diff
@@ -139,6 +139,19 @@ export default configMerger(walttiConfig, {
     ],
   },
 
+  sourceForAlertsAndDisruptions: {
+    VARELY: {
+      fi: 'Seutu+',
+      sv: 'Seutu+',
+      en: 'Seutu+',
+    },
```
Comment (optionsome) [inline app/configurations/config.varely.js]: Same as above.

**Review by optionsome — CHANGES_REQUESTED** [review]
Btw I wonder if there is any reason for us not to include Turku's realtime now as well now in seutuplus?

**Comment by sharhio** [comment]
There is no reason to not add Turku, in fact, it should be added.

**Pull request merged**